packages/devtools/src/commands/elementClear.js

```diff
@@ -0,0 +1,13 @@
+import command from '../scripts/elementClear'
+
+export default async function elementClear ({ elementId }) {
+    const elementHandle = this.elementStore.get(elementId)
+
+    if (!elementHandle) {
```

Comment:
if it's not in cache, should we fetch it?

Reply:
No, this functionality is handled in WebdriverIO. We should stay as close to the WebDriver protocol as possible.